### PR TITLE
preparing the website to 2019 summit!

### PR DESCRIPTION
**Pull request by neta-kedem**
preparing the website to 2019 summit
adjust the cfp to the new design

### CODE REVIEW

File: client/src/utils.js [hunk 1]
```diff
@@ -13,8 +13,8 @@ export const colors = ['purple', 'gold', 'cyan', 'blue', 'pink', 'yellow', 'gree
 export const isServer = window === '__server';
 export const history = isServer ? createMemoryHistory() : createBrowserHistory();
 
-export const REVERSIM_SUMMIT = 'Reversim Summit 2018';
-export const REVERSIM_MAIL = 'rs18-team@googlegroups.com';
+export const REVERSIM_SUMMIT = 'Reversim Summit 2019';
+export const REVERSIM_MAIL = 'rs19-team@googlegroups.com';
```
Comment (rantav) [inline client/src/utils.js]: Actually it's `rs19team@googlegroups.com`

**Comment by rantav** [comment]
FYI @shlomi-noach and @amitzur if you have cycle to review this. 

@neta-kedem unfortunately I don't have the capacity to review now (family matters...) so feel free to wait a day or two for others to review and if none then merge.

gracias

File: client/src/components/Footer.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const Footer = ({user}) => (
     <Container>
       <Row className="footer-content align-items-center" noGutters>
         <div className="d-flex">
-          <a className="text-white mr-6" href="mailto:rs18-team@googlegroups.com">
+          <a className="text-white mr-6" href="mailto:rs19-team@googlegroups.com">
```
Comment (shlomi-noach) [inline client/src/components/Footer.js]: `rs19team`:

```suggestion
          <a className="text-white mr-6" href="mailto:rs19team@googlegroups.com">
```

File: client/src/components/Hero.js [hunk 3]
```diff
@@ -21,11 +21,14 @@ const Hero = () => (
...
-            <Button>View Schedule</Button>
+          {/*<Link to="/schedule">*/}
+            {/*<Button>View Schedule</Button>*/}
+          {/*</Link>*/}
+          <Link to="/cfp">
+            <Button> Submit session </Button>
           </Link>
         </div>
-        <div className={cn(subtitle, 'mb-8')}>8 - 9 October | Tel Aviv University</div>
+        <div className={cn(subtitle, 'mb-8')}>17 - 18 June | HaCameri Tel Aviv</div>
```
Comment (shlomi-noach) [inline client/src/components/Hero.js]: OK for now -- this is subject to change

File: client/src/components/Location.js [hunk 4]
```diff
@@ -9,15 +9,18 @@ const Location = () => (
         Venue
       </h1>
       <div className="bg-emph py-9 px-4 d-inline-block h1-up">
-        <h3 className="mb-0">Tel Aviv University, Smolarz Auditorium</h3>
+        <h3 className="mb-0">HaCameri theater,<br/>
+          Sderot Sha'ul HaMelech 19
```
Comment (shlomi-noach) [inline client/src/components/Location.js]: OK for now, this is subject to change

File: client/src/data/proposals.js [hunk 5]
```diff
@@ -24,9 +24,9 @@ export const PROPOSAL_TYPES_ARR = Object.keys(PROPOSAL_TYPES).map(type => ({
   text: PROPOSAL_TYPES[type],
 }));
 
-export const CFP_ENDS = new Date(2018, 4, 31);
-export const CFP_ENDS_STR = 'May 31st midnight UTC';
-export const CFP_ENDS_STR_SHORT = 'May 31st';
+export const CFP_ENDS = new Date(2019, 1, 20);
+export const CFP_ENDS_STR = 'February 20st midnight UTC';
+export const CFP_ENDS_STR_SHORT = 'Feb 20';
```
Comment (shlomi-noach) [inline client/src/data/proposals.js]: OK for now, this is subject to change

File: client/src/data/timeline.js [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
-  {time: 'June 10', text: 'Public voting ends'},
-  {time: 'Mid July', text: 'Agenda published'},
-  {time: 'Mid September', text: 'Open registration'},
-  {time: 'October 8-9', text: 'Reversim Summit 2018'},
+  {time: 'Feb 22', text: 'Call for papers ends'},
+  {time: 'Mar 1', text: 'Public voting begins'},
+  {time: 'Mar 10', text: 'Public voting ends'},
+  {time: 'Mid April', text: 'Agenda published'},
+  {time: 'Mid May', text: 'Open registration'},
+  {time: 'June 17-18', text: 'Reversim Summit 2019'},
```
Comment (shlomi-noach) [inline client/src/data/timeline.js]: subject to change

File: server/init/eventConfig.js [hunk 7]
```diff
@@ -1,10 +1,10 @@
...
 
 const config = {
-  cfpStartDate: process.env.CFP_START_DATE ||  "2018-04-25",
-  cfpEndDate: process.env.CFP_END_DATE || "2018-05-31",
-  votingStartDate: process.env.VOTING_START_DATE || "2018-06-01",
-  votingEndDate: process.env.VOTING_END_DATE || "2018-06-10",
+  cfpStartDate: process.env.CFP_START_DATE ||  "2019-01-20",
+  cfpEndDate: process.env.CFP_END_DATE || "2019-02-20",
+  votingStartDate: process.env.VOTING_START_DATE || "2019-04-01",
+  votingEndDate: process.env.VOTING_END_DATE || "2019-05-01",
```
Comment (shlomi-noach) [inline server/init/eventConfig.js]: subject to change

File: client/src/utils.js (hunk 1 above)
Comment (shlomi-noach) [reply client/src/utils.js]: @rantav are you familiar with "suggested changes"? https://twitter.com/github/status/1053360435074949120

Comment (rantav) [reply client/src/utils.js]: Fix: 

```suggestion
export const REVERSIM_MAIL = 'rs19team@googlegroups.com';
```

there I used it ;-)

**Pull request merged**